config/jobs/kubernetes/test-infra/test-infra-canaries.yaml

```diff
@@ -53,3 +53,40 @@ periodics:
     testgrid-dashboards: sig-testing-canaries
     testgrid-tab-name: multiple-container-test
     description: echo from two different containers
+- interval: 1h
+  name: ci-test-infra-larger-multiple-container-test
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  labels:
+    preset-service-account: "true"
+    preset-bazel-scratch-dir: "true"
+  spec:
+    containers:
+    - name: benchmark-junit-container
+      image: gcr.io/k8s-testimages/benchmarkjunit:latest
+      command:
+      - /benchmarkjunit
+      args:
+      - --log-file=$(ARTIFACTS)/benchmark-log.txt
+      - --output=$(ARTIFACTS)/junit_benchmarks.xml
+      - --pass-on-error
+      - ./experiment/dummybenchmarks/...
+    - name: bazel-test-container
+      image: gcr.io/k8s-testimages/launcher.gcr.io/google/bazel:v20200713-e9b3d9d-test-infra
+      command:
+      - hack/bazel.sh
+      args:
+      - test
+      - --config=ci
+      - --nobuild_tests_only
+      - //...
+  annotations:
+    testgrid-alert-email: anthonytong@google.com
+    ProwMultipleContainerSupport: Yes, I know what I'm doing.
```

Comment:
This is no longer required, right?

Reply:
Yeah, that PR to remove the annotation is merged now.  I'll remove it for both of the two jobs.